console/frontend/src/main/frontend/src/app/components/pages/pages-navigation/pages-navigation.component.html

```diff
@@ -18,7 +18,7 @@
       <app-custom-views></app-custom-views>
 
       <li routerLinkActive="active">
-        <a routerLink="/status"><i class="fa fa-dashboard"></i> <span class="nav-label">Adapter Status</span></a>
+        <a data-cy-main="adapterStatus" routerLink="/status"><i class="fa fa-dashboard"></i> <span class="nav-label">Adapter Status</span></a>
```

Comment:
Ik laat de definitieve beslissing aan Vivy, maar ik zou hier `data-cy-nav` verwachten ipv main.

Reply:
`data-cy-main` is inderdaad niet echt een duidelijke beschrijving, Omdat het gaat om het navigeren naar een andere Route binnen Angular zou ik eerder kiezen voor `data-cy-nav` of als je het technischer wil hebben kan je het `data-cy-route` noemen.

Reply:
Verwerkt.